packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.constants.js

```diff
@@ -1,5 +1,17 @@
 export const SOFTPHONE_TYPE = 'softphone';
+export const MOBILE_OS = {
+  windows: 'Windows Phone',
+  android: 'Android',
+  ios: 'iOS',
+};
 
+export const ALT_SOFTPHONE_STORE_LINK = {
+  windows: 'telephony_line_softphone_store_link_windows',
+  android: 'telephony_line_softphone_store_link_android',
+  ios: 'telephony_line_softphone_store_link_ios',
+};
```

Comment:
**nitpick:** No need to maintain it in constants file. Directly use the required string in HTML.